charts/streams-app/templates/deployment.yaml

```diff
@@ -46,6 +46,12 @@ spec:
         {{ $key }}: {{ $value }}
         {{- end }}
     spec:
+      {{- if .Values.tolerations.key }}
```

Comment:
tolerations should be a list. Maybe you need to do it similar to resources where you just put the yaml as is

Reply:
Should we still support default values for `operator` and `effect`, so that we can do something like:
```
tolerations:
  - key1
  - key2
      operator: Exists
      effect: NoExecute
  - key3
```

Reply:
Just let the user put there what ever he would like in order to be fully flexible wrt tolreations. No defaults or sth like that